tests/generators/bls/main.py

```diff
@@ -51,6 +53,7 @@ def hex_to_int(x: str) -> int:
 ]
 PUBKEYS = [bls.SkToPk(privkey) for privkey in PRIVKEYS]
 
+ZERO_PUBKEY = b'\x00' * 48
 Z1_PUBKEY = b'\xc0' + b'\x00' * 47
 NO_SIGNATURE = b'\x00' * 96
```

Comment:
not a strong suggestion but consistency w/ `ZERO_PUBKEY / Z1_PUBKEY` and `NO_SIGNATURE / Z2_SIGNATURE` would be nice -- so change `NO_SIGNATURE` to `ZERO_SIGNATURE` (or change `ZERO_PUBKEY` to `NO_PUBKEY` but that is less clear imo)

Reply:
Good point 👍